test/e2e/framework/scaletest/add-shared-labels.go

```diff
@@ -12,6 +13,10 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+const (
+	timeoutToLabelAllPodsMinutes = 120
```

Comment:
Again, please use `time.Duration`:

```suggestion
	podLabelTimeout = 120 * time.Minute
```